tests/ast/conftest.py

```diff
@@ -0,0 +1,158 @@
+#
+# Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
+#
+import logging
+import os
+from functools import cached_property
+
+import pytest
+
+from snowflake.snowpark import Session
+
+
+def default_unparser_path():
+    explicit = os.getenv("SNOWPARK_UNPARSER_JAR")
+    default_default = f"{os.getenv('HOME')}/Snowflake/trunk/Snowpark/unparser/target/scala-2.13/unparser-assembly-0.1.jar"
+    return explicit or default_default
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--unparser-jar",
+        action="store",
+        default=default_unparser_path(),
+        type=str,
+        help="Path to the Unparser JAR built in the monorepo. To build it, run `sbt assembly` from the unparser directory.",
+    )
+    parser.addoption(
+        "--update-expectations",
+        action="store_true",
+        default=False,
+        help="If set, overwrite test files with the actual output as the expected output.",
+    )
+
+
+def pytest_configure(config):
+    pytest.unparser_jar = config.getoption("--unparser-jar")
+    if not os.path.exists(pytest.unparser_jar):
```

Comment:
the check here changed in the recent phase0 branch, can we update? I.e., we only error out when update_expectations is true and no unparser jar is found.